src/pages/pass-finalisation/cat-adi-part2/_tests_/pass-finalisation.cat-adi-part2.page.spec.ts

```diff
@@ -28,11 +28,14 @@ import { WarningBannerComponent } from '../../../../components/common/warning-ba
 import { FormGroup, FormControl, Validators } from '@angular/forms';
 import { Subscription } from 'rxjs';
 import { configureTestSuite } from 'ng-bullet';
+import { CAT_ADI_PART2 } from '../../../page-names.constants';
 
 describe('PassFinalisationCatADIPart2Page', () => {
   let fixture: ComponentFixture<PassFinalisationCatADIPart2Page>;
   let component: PassFinalisationCatADIPart2Page;
   let store$: Store<StoreModel>;
+  let navController$: NavController;
```

Comment:
$ suffix convention signifies that the variable is an observable can you remove here